python/cuml/fil/fil.pyx

```diff
@@ -137,6 +137,12 @@ cdef class TreeliteModel():
         model.set_handle(handle)
         return model
 
+    @staticmethod
+    def from_model_handle(treelite_handle):
```

Comment:
I would call this `from_treelite_model_handle` since potentially other model types could have handles.